### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by korshaknn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.7.1
+
+### Vision
+- Fixed dynamic `FPS` issue for a `Merge model`
+- Fix problem with neural networks run (swap `RGB channels`)
```
Comment (yunikkk) [inline CHANGELOG.md]: I'd rephrase, 
```
- Fixed ModelPerformanceMode.DYNAMIC not taking effect with ModelPerformanceConfig.Merged
- Fixed detections of sign objects
```
so that it's clear to end user

Comment (korshaknn) [reply CHANGELOG.md]: done
